package/graph.py

```diff
@@ -17,14 +18,13 @@ def display(snippet_name, password):
     )
 
     try:
-        with open(snippet_path, "r") as file:
-            source_code = file.read()
-
-     
+        base_dir = os.path.dirname(__file__)
+        snippet_path = os.path.join(base_dir, "stash", f"{snippet_name}.py")
```

Comment:
Duplicate line, its already defined at line 16.

Reply:
okay ok got it. let me try.